pennychat/models.py

```diff
@@ -66,13 +67,15 @@ def __repr__(self):
 
 
 class Participant(models.Model):
-    ORGANIZER = 1
-    ATTENDEE = 2
-    INVITEE = 3
+    ORGANIZER = 10
+    ATTENDEE = 20  # Not necessarily invited by name, could have come from a channel invite
```

Comment:
For some reason, this comment is confusing to me. I get what you are saying, but it makes it seem like "ATTENDEE" is some sort of other form of "INVITEE", when really it's just "anyone who has _accepted_ RSVPed". Low FG score for actually changing the comment, but I figured I'd let you know how I read it (and how a future fellow coder may read it? 😄).

Reply:
I updated it a bit.